### PR TITLE
Maryia/Feat: Added new CFD Personal details modal for DBVI

### PR DESCRIPTION
**Pull request by maryia-deriv**
## Changes:

Please include a summary of the change and which issue is fixed below:
-   Added new CFD Personal details modal for DBVI

## When you need to add unit test

-   [ ] If this change disrupt current flow
-   [ ] If this change is adding new flow

## When you need to add integration test

-   [ ] If components from external libraries are being used to define the flow, e.g. @deriv/components
-   [ ] If it relies on a very specific set of props with no default behavior for the current component.

## Test coverage checklist (for reviewer)

-   [x] Ensure utility / function has a test case 
-   [x] Ensure all the tests are passing

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: packages/cfd/src/Components/__tests__/cfd-personal-details-form.spec.js [hunk 1]
```diff
@@ -158,7 +158,7 @@ describe('<CFDPersonalDetailsForm />', () => {
     });
 
     it('should show an error message received from server that is passed via props as form_error', () => {
-        const form_error = 'Form submission failed.';
+        const form_error = 'Input validation failed: citizen!';
```
Comment (matin-deriv) [inline packages/cfd/src/Components/__tests__/cfd-personal-details-form.spec.js:161]: where is this message coming from?

Comment (maryia-deriv) [reply packages/cfd/src/Components/__tests__/cfd-personal-details-form.spec.js:161]: @matin-binary it's coming from BE.  Actually we validate the fields using Formik before we send the data to BE so this particular message is hardly ever going to be shown to a user. Would you like me to leave the message as it was before with a random 'Form submission failed.' text ?🙂

File: packages/cfd/src/Containers/jurisdiction-modal-content.tsx [hunk 2]
```diff
@@ -315,7 +317,7 @@ const JurisdictionModalContent = ({
                         line_height='xs'
                         className='cfd-jurisdiction-card____jurisdiction-checkbox--description'
                     >
-                        I confirm and accept Deriv Investments (Europe) Limited's Terms and Conditions
+                        I confirm and accept Deriv Investments (Europe) Limited&#39;s Terms and Conditions
```
Comment (maryia-deriv) [inline packages/cfd/src/Containers/jurisdiction-modal-content.tsx:320]: fixed it to eliminate no-unescaped-entities error that remained in the main branch

File: packages/cfd/src/Containers/jurisdiction-modal-content.tsx [hunk 3]
```diff
@@ -157,16 +157,18 @@ const JurisdictionCard = ({
                     />
                 </h2>
 
-                {jurisdiction_contents[type_of_card as keyof typeof jurisdiction_contents].contents.map(item => (
-                    <div className='cfd-jurisdiction-card__bullet-wrapper'>
-                        <div>
-                            <Checkmark />
+                {jurisdiction_contents[type_of_card as keyof typeof jurisdiction_contents].contents.map(
+                    (item, index) => (
+                        <div className='cfd-jurisdiction-card__bullet-wrapper' key={index}>
```
Comment (maryia-deriv) [inline packages/cfd/src/Containers/jurisdiction-modal-content.tsx:162]: fixed it to eliminate jsx-key error that remained in the main branch

File: packages/cfd/src/Components/__tests__/cfd-personal-details-form.spec.js (hunk 1 above)
Comment (matin-deriv) [reply packages/cfd/src/Components/__tests__/cfd-personal-details-form.spec.js:161]: it is okay as long as coming from BE.

**Pull request merged**